irispy/iris_tools.py

```diff
@@ -98,15 +109,14 @@ def get_iris_response(pre_launch=False, response_file=None, response_version=Non
         dn2phot_sji:  `tuple` of length 4
         comment: `str`
         version: `int`
-        version_date: `datetime.datetime`
+        version_date: `time.time`
```

Comment:
`astropy.time.Time`

Reply:
Yes.  All instances of ```"`time.time`"``` should be changed to ```"`astropy.time.Time`"```